kostyor/db/api.py

```diff
@@ -10,24 +12,34 @@
 from sqlalchemy.orm import scoped_session, sessionmaker
 
 
-# TODO (sc68cal) save the database file in a configurable location
-engine = create_engine('sqlite:////tmp/test.db', convert_unicode=True)
-db_session = scoped_session(sessionmaker(autocommit=False,
-                                         autoflush=False,
-                                         bind=engine))
+@contextmanager
+def create_db_session():
+    """ Creates a context with an open SQLAlchemy session.
+    """
+    engine = create_engine(conf.CONF.database, convert_unicode=True)
```

Comment:
Generally it's not a good idea to create engine multiple times. It should be created only once. It manages connections pool and it'd be nice to benefit from it.